src/main/version.ts

```diff
@@ -1 +1 @@
-export const SDK_VERSION = '2.5.4';
+export const SDK_VERSION = '3.0.0-beta.2';
```

Comment:
Do we have to do this?

Reply:
I should actually remove this file from source control, add it to `.gitignore` and generate it on every build!